### ISSUE TITLE
After a certain number of "SPEAK", it stops... Is there a limit to the number of words "tts.speak" calls that we can make ? 

### ISSUE THREAD

**Issue by sylvainjack** [dlutton/flutter_tts#308]
### Expected behavior
I expected it to have no limit...
### Configuration
I check it on an iPhone XS PRO MAX. 
**Version:** 0.1.x

**Platform:** 
- [ X] :iphone: iOS
- [ ] :robot: Android

**Comment by dlutton** [comment]
@sylvainjack sorry for the delay, but I haven't used this with speech_to_text, do you have a small sample app with code where I can play around with it to determine if the issue is on the end of this plugin?

**Comment by sylvainjack** [comment]
Oops, no... I am just working on one app, which is quite large... Maybe adding the speech_to_text to the TTS example app ? to see ?

**Comment by dlutton** [comment]
@sylvainjack good point, I'll try with the example.  Also, were there any logs from the bug you encountered or did it just simply stop speaking?

**Comment by sylvainjack** [comment]
No, no logs, it just stopped speaking.

**Comment by Roshanalexander** [comment]
Seeing the same issue.

**Comment by sylvainjack** [comment]
I have run some more tests concerning the interference between TTS and Speech_to_text : the bug is really systematic : User click on "speaker" to hear the word (with TTS) : it speaks fine... user clicks on "MICROPHONE ICON" (To translate the word he heard, it's a vocabulary app) : if user waits one or two seconds after TTS is finished, everything is fine. If user clicks on MIKE while TTS is speaking or right after TTS finished, then TTS is terminated and will no longer speak. Even a restart won't work, have to recompile the whole app. 
Have you found any idea why this happened ? 
TTS seems to interfere with AUDIOPLAYERS package as well... for some reason, AUDIOPLAYERS won't really work when I use TTS. (In my app : I have a sound effect being played when user gets the word right : if he translates right, we hear a sound, if he makes a mistake, we hear a sound). Well, it only works once.
